src/NuGet.Core/NuGet.PackageManagement/PublicAPI.Shipped.txt

```diff
@@ -362,7 +362,7 @@ NuGet.ProjectManagement.BuildIntegratedInstallationContext.SuppressParent.get ->
 NuGet.ProjectManagement.BuildIntegratedInstallationContext.SuppressParent.set -> void
 NuGet.ProjectManagement.BuildIntegratedInstallationContext.UnsuccessfulFrameworks.get -> System.Collections.Generic.IEnumerable<NuGet.Frameworks.NuGetFramework>
 NuGet.ProjectManagement.BuildIntegratedPackageReference
-NuGet.ProjectManagement.BuildIntegratedPackageReference.BuildIntegratedPackageReference(NuGet.LibraryModel.LibraryDependency dependency, NuGet.Frameworks.NuGetFramework projectFramework) -> void
+NuGet.ProjectManagement.BuildIntegratedPackageReference.BuildIntegratedPackageReference(NuGet.LibraryModel.LibraryDependency dependency, NuGet.Frameworks.NuGetFramework projectFramework, NuGet.Packaging.Core.PackageIdentity installedVersion) -> void
```

Comment:
Granted this is in the only assembly we're more liberal in allowing breaking changes. But in general, you should not be changing shipped APIs. Instead, create a new overload, and change the existing method to call the new overload with an appropriate default value for the new parameter.

Any customers using our package in their own app and using this API will be broken if this change ships as-is.